snowpack/src/build/import-resolver.ts

```diff
@@ -22,7 +22,10 @@ export function getFsStat(importedFileOnDisk: string): fs.Stats | false {
 }
 
 /** Resolve an import based on the state of the file/folder found on disk. */
-function resolveSourceSpecifier(lazyFileLoc: string, config: SnowpackConfig) {
+function resolveSourceSpecifier(
+  lazyFileLoc: string,
+  {parentFile, config}: {parentFile: string; config: SnowpackConfig},
```

Comment:
We need the `parentFile` location in case extension is missing, to try that extension first